solutions/sequences/ValidAnagram.cpp

```diff
@@ -0,0 +1,34 @@
+// https://leetcode.com/problems/valid-anagram/
+
+#include <vector>
+
+#define NUM_OF_LETTERS 26
+
+class Solution {
+public:
+    bool isAnagram(string s, string t) {
+        int sLettersCount[NUM_OF_LETTERS] = { 0 };
```

Comment:
looks good.  There is also a question of "Group Anagrams" that I recommend you visit later